lms/config.py

```diff
@@ -41,6 +48,8 @@ def __post_init__(self):
 
 
 SETTINGS = (
+    # List of users that are ADMINS in the admin pages
+    _Setting("admin_users", value_mapper=aslist),
```

Comment:
We discussed having this backed but a DB table but while that option is not that complicated this is so much simpler.

I don't expect a lot of changes to this value in the prod environments over time.